crates/bevy_app/src/app.rs

```diff
@@ -1068,4 +1103,60 @@ mod tests {
         }
         App::new().add_plugin(PluginRun);
     }
+
+    mod system_stepping {
```

Comment:
Strong preference to have this live in its own file: this file is already way too big.

Reply:
Is `crates/bevy_app/src/tests.rs` the right place?  Or does this need to be broken out into an `crates/bevy_app/src/app/...` directory layout?

Reply:
The former will work just fine. It can be our first "integration test" for this crate.

Reply:
Ended up moving the stepping tests under `crates/bevy_app/tests/stepping_test.rs`; it matched more what I was seeing about integration tests for rust, and I didn't have to throw a `#[cfg(tests)]` before `mod tests` in `lib.rs`.

Reply:
Ah, I totally misread. Yes, that was my hope!